ui/src/testing/react-test-helpers.ts

```diff
@@ -0,0 +1,9 @@
+import {ReactWrapper} from 'enzyme';
+
+
+// This file is necessary because angular imports complain if there
```

Comment:
What is the exact error you were seeing? AFAICT your `.spec.tsx` test should not have any dependency on Angular (though I guess there are some related imports in the file). Could you expand this comment and include when you intend a client to use this?

Also I definitely don't understand why there are two identical promise invocations below, a comment there would be good.

Reply:
It complained because there was no definition of `Zone`, which meant the imports failed.

Reply:
If I imported from the main test file.